stl/inc/xloctime

```diff
@@ -539,6 +539,10 @@ protected:
 private:
     ios_base::iostate __CLRCALL_OR_CDECL _Getint(_InIt& _First, _InIt& _Last, int _Lo, int _Hi, int& _Val,
         const _Ctype& _Ctype_fac) const { // get integer in range [_Lo, _Hi] from [_First, _Last)
+        const int _NHiDigits =

```

Comment:
Naming nitpick: The codebase isn't entirely consistent, but we generally use `_UglyCamelCase` for types (like `_InIt`, `_FwdItHaystack`, etc.) and `_Ugly_snake_case` for variables. Here, I think `_Hi_digits` would be fine.

Reply:
I will try to follow this convention on my next PRs.